CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ See also https://docs.resim.ai/changelog/ for all ReSim changes
 
 Changes in this section will be included in the next release.
 
+#### Added
+
+- Allowed the ability to persist project selection with `select project --project foo". If set via this method, `--project` will no longer be a required flag for other commands.
```

Comment:
```suggestion
- Allowed the ability to persist project selection with `select project --project foo`. If set via this method, `--project` will no longer be a required flag for other commands.
```

Could this be `select project <project name>` i.e. avoiding the repetition of project or do we think the inconsistency (flag vs no flag) would be annoying?